rsconf/component/docker.py

```diff
@@ -83,6 +78,33 @@ def internal_build(self):
             run_d=z.run_d,
         )
 
+    def _install_daemon_json(self, z):
+        p = self.tmp_path()
+        d = PKDict(
```

Comment:
Since we are on the subject of copies, it's much better to not use the copy and do an update, or better:

```py
PKDict({
    "hosts": ...
    ...
    "data-root": ...
}).update(z.get("daemon_aux", {}))
```

While it's not a performance issue per se, these types of "habits" are not good to have in production code.

Reply:
1bd52a8